ztp/ztp-ran-policy-generator/kustomize/plugin/ranPolicyGenerator/v1/ranpolicygenerator/policyGen/policyBuilder.go

```diff
@@ -25,23 +25,38 @@ func (pbuilder *PolicyBuilder) Build(customResourseOnly bool) (map[string]interf
 		namespace, path, matchKey, matchValue, matchOper := pbuilder.getPolicyNsPath()
 		subjects := make([]utils.Subject , 0)
 		for id, sFile := range pbuilder.RanGenTemp.SourceFiles {
+
 			pname, rname := pbuilder.getPolicyName(id)
+			name := pname + "-" + sFile.PolicyName
+			err := CheckNameLength(namespace, name)
```

Comment:
Where does this "namespace + name" get used. Maybe the check can be moved there? It is a bit confusing to see a check here but I don't see where the "namespace+name" is used.

Reply:
I'm not sure why its confusing, based on the url in the PR comment the policy namespace + name must not exceed 63 chars so we check the length before start creating the policy. 
namespace: is common-ran-sub|{group-name}-ran-sub|{site-name}-ran-sub based on RanGenTemp.metadata.labels
name consist of:  prefix name which is common|groupName|siteName + "-" + policyName that is given by the user

Reply:
what url?

Reply:
In the PR comment 
[0] https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.2/html/security/governance-and-risk